diesel/src/pg/expression/expression_methods.rs

```diff
@@ -3749,4 +3750,14 @@ pub(in crate::pg) mod private {
     impl TextArrayOrNullableTextArray for Array<Nullable<Text>> {}
     impl TextArrayOrNullableTextArray for Nullable<Array<Text>> {}
     impl TextArrayOrNullableTextArray for Nullable<Array<Nullable<Text>>> {}
+
+    #[diagnostic::on_unimplemented(
+        message = "`{Self}` is neither `Record<Text>`, `Record<Nullable<Text>>`,\
+                   `Nullable<Record<Text>>` nor `diesel::sql_types::Nullable<Record<Nullable<Text>>>`",
+        note = "try to provide an expression that produces one of the expected sql types"
+    )]
```

Comment:
The reference to `Text` fragments here is outdated

Reply:
Opps! My bad, guess I missed to unstash some changes, I'll fix this.